data-raw/DATASET.R

```diff
@@ -365,14 +365,6 @@ lymphoma_genes = dplyr::select(lymphoma_genes,-entrezgene_id) %>% group_by(Gene,
 #lymphoma_genes_comprehensive = dplyr::filter(lymphoma_genes_comprehensive,Reddy==TRUE | Chapuy == TRUE | LymphGen == TRUE | curated == TRUE)
 #lymphoma_genes_comprehensive$other_support = ""
 
-# Example SSM data from Grande et. al, 2019
-grande_maf = system.file("extdata", "blood8871418-suppl2-ssm.csv", package = "GAMBLR.data") %>%
-  read_tsv(.) %>%
-  as.data.frame
-grande_maf = grande_maf[!colnames(grande_maf) %in% c("tumor_biospecimen_id","normal_biospecimen_id")]
-
-usethis::use_data(grande_maf, overwrite = TRUE)
```

Comment:
I might miss that, but why `grande_maf` was deleted?

Reply:
With the introduction of the new dataset, called [sample_data](https://github.com/morinlab/GAMBLR.data/blob/master/man/sample_data.Rd), the Grande MAF is redundant in many ways. This new dataset has metadata as well as maf, seg, and bedpe data in both projections bundled.

Reply:
Understood. Thanks for the explanation.